get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,160 @@
+---
+title: What is Shiny?
+---
+
+```{=html}
+<style>
+  .btn-links {
+    display: flex;
+    gap: 1.5rem;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .btn-link-large {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 150px;
+    height: 150px;
+    font-size: 1.5rem;
+    border: 1px solid transparent;
+    border-radius: 10px;
+    cursor: pointer;
+    padding: 1.25rem;
+    text-decoration: none;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  }
+
+  .btn-link-large i {
+    margin-top: 10px;
+    font-size: 2rem;
+  }
+
+  .btn-link-large:hover {
+    text-decoration: none;
+    border: 1px solid var(--bs-primary);
+  }
+</style>
+```
+
+Whether you’re a data scientist, analyst, or developer, Shiny makes it easy to [create](create-run.qmd) rich, [interactive experiences](/docs/ui-overview.qmd) in pure Python [without needing](/docs/reactive-foundations.qmd) to learn JavaScript or front-end frameworks.
```

Comment:
Links should always make sense if read entirely out of context. Here "interactive experiences" links to UI (makes sense), but "without needing" links to reactive foundations. Always make sure that if you read only the link text, the place where it takes you makes sense.

In this case, I think you could either drop the link or work in the word "reactive".